newIDE/app/src/KeyboardShortcuts/DefaultShortcuts.js

```diff
@@ -3,6 +3,7 @@ import { type CommandName } from '../CommandPalette/CommandsList';
 
 export type ShortcutMap = { [CommandName]: string };
 
+
```

Comment:
When you're creating a future PR, make sure to remove the useless lines like this from your PR. For this, proofread your PR before sending it for review and add commits that undo these changes. You can learn more by searching for git/github desktop.

Reply:
Ahh I see. Thank you.